SerialUART/CS/README.md

```diff
@@ -0,0 +1,462 @@
+
+# Serial UART 
+
+We'll create a simple app that allows communication between a desktop and an IoT device over a serial interface.
+
+### Load the project in Visual Studio
+
+Make a copy of the folder on your disk and open the project from Visual Studio 2019.
+
+This app is a Universal Windows app and will run on both the PC and your IoT device.
+
+### Wiring the serial connection 
+
+You have two options for wiring up your board:
+
+1. using the On-board UART controller
+2. using a USB-to-TTL adapter cable such as [this one](http://www.adafruit.com/products/954).
+
+#### <a name="UpBoard_UART"></a>On-board UART (UpBoard)
+
+The UpBoard has an on-board UART. See the [UpBoard pin mapping page](../../Resources/PinMappingsUpBoard.png) for more details on the UpBoard GPIO pins. 
+
+* UART uses GPIO pins 6, 8, 10  
+
+Make the following connections:
+
+* Insert the USB end of the USB-to-TTL cable into a USB port on the PC
+* Connect the GND wire of the USB-to-TTL cable to Pin 6 (GND) on the Upboard

```

Comment:
Upboard is spelled Upboard here, but Up-Board or UpBoard elsewhere in this document.  Please make consistent.

Reply:
Changed the Up-Board to Upboard in the readme.

Reply:
It should be "Up Board" - https://up-board.org/up/specifications/

Reply:
I'm mentioning it as Upboard and keeping it consistent in all the readmes, Hope it is fine?

Reply:
Please change it to "UP Board".